gneiss/_summary.py

```diff
@@ -53,12 +51,14 @@ def __init__(self,
             p = r.pvalues
             p.name = r.model.endog_names
             self.pvalues = self.pvalues.append(p)
-            sse += r.ssr
-            ssr += r.ess
 
+    @property
+    def r2(self):
+        ssr = sum([r.ssr for r in self.results])
+        sse = sum([r.sse for r in self.results])
         # calculate the overall coefficient of determination (i.e. R2)
         sst = sse + ssr
-        self.r2 = 1 - sse / sst
+        return 1 - sse / sst
```

Comment:
Nope.  This is handled within the ols results.  Note that I wanted to move this outside of the constructor, because linear mixed effects models don't have an r2.